pkg/controller/defaults/resources.go

```diff
@@ -79,10 +79,20 @@ var (
 				corev1.ResourceMemory: resource.MustParse("4Gi"),
 			},
 		},
-		"noobaa-db-vol": corev1.ResourceRequirements{
+		"noobaa-db-vol": {
 			Requests: corev1.ResourceList{
 				corev1.ResourceStorage: resource.MustParse("50Gi"),
 			},
 		},
+		"noobaa-endpoint": {
+			Requests: corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("1"),
+				corev1.ResourceMemory: resource.MustParse("2Gi"),
+			},
+			Limits: corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("1"),
+				corev1.ResourceMemory: resource.MustParse("2Gi"),
+			},
+		},
```

Comment:
If the above changes are valid, move this to a separate commit.

Reply:
It is in a separate commit, Do you mean a separate PR?

Reply:
The issue is that the gofmt changes are unrelated to the NooBaa endpoint changes. The gofmt changes should go into their own commit, with a message explaining why they're happening.